.github/workflows/labeler.yml

```
@@ -0,0 +1,56 @@
+              });
+            }
+              } catch (error) {
+              core.setFailed(`Failed to update PR: ${error.message}`);
+            }
+
+  labeler:
+    name: "Add labels"
+    needs: coderabbit
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea # v7.0.1
```

yamllint

[warning] too few spaces before comment

(comments)